example/performance/README.md

```diff
@@ -0,0 +1,106 @@
+# Performance
+
+Palisade includes a performance tool for testing some simple scenarios. It uses the example data and generates some fake HR data using
+the [HR data generator](../hr-data-generator/README.md).
+
+## Usage
+
+### Building
+To use the performance tool, you must first build Palisade (from the Palisade root directory):
+```bash
+mvn clean install -P example
+./example/deployment/local-jvm/bash-scripts/buildServices.sh
+```
+
+Open a separate terminal window and start the Palisade services running on your local machine:
+```bash
+./example/deployment/local-jvm/bash-scripts/startAllServices.sh
+```
+Wait for those to initialize and then from the first terminal you can run:
+```bash
+./example/deployment/bash-scripts/perf.sh
+```
+
+### Setting up
+Without arguments, the tool will give some usage information.
+
+The first step is to create some data for the tool to test against. For the performance tests, we use a "small" file and a "large" file.
+You can decide the number of HR records to go in the small and large files (depending on how long you want the tests to run!), or even
+make them the same size!
+
+First, we tell Palisade where its bootstrap configuration file is:
+```bash
+export PALISADE_REST_CONFIG_PATH=./example/example-model/src/main/resources/configRest.json
+```
+
+Now, to create some test data in a `perf_test` directory:
+
+```bash
+mkdir perf_test
+./example/deployment/bash-scripts/perf.sh 1000 10000 perf_test
+```
+
+This will produce output similar to:
+```bash
+example.perf.Perf INFO  - Going to create 1000 records in file employee_small.avro and 10000 records in file employee_large.avro in sub-directory
+example.perf.Perf INFO  - Creation tasks submitted...
+example.perf.Perf INFO  - Small file written successfully true
+example.perf.Perf INFO  - Large file written successfully true
+example.perf.Perf INFO  - Copying small file
```

Comment:
Worth having some explanation for why it is copying files, this might be in an introduction paragraph, explaining what conditions the performance tools are testing

Reply:
Done